### ISSUE TITLE
Context is not defined (push)

### ISSUE THREAD

**Issue by HendrikRoth** [Strider-CD/strider-docker-build#19]
It seams, that the context is not defined, called in https://github.com/Strider-CD/strider-docker-build/blob/master/lib/push.js#L21

**Comment by oliversalzburg** [comment]
It should probably be passed here: https://github.com/Strider-CD/strider-docker-build/blob/master/lib/build.js#L44

**Comment by HendrikRoth** [comment]
```
context is not defined

ReferenceError: context is not defined
    at null.<anonymous> (/usr/lib/node_modules/strider/node_modules/strider-docker-build/lib/push.js:23:11)
    at emit (events.js:95:17)
    at null.<anonymous> (_stream_readable.js:765:14)
    at emit (events.js:92:17)
    at emitReadable_ (_stream_readable.js:427:10)
    at emitReadable (_stream_readable.js:423:5)
    at readableAddChunk (_stream_readable.js:166:9)
    at Readable.push (_stream_readable.js:128:10)
    at Transform.push (_stream_transform.js:140:32)
    at JSONStream._transform (/usr/lib/node_modules/strider/node_modules/strider-docker-build/node_modules/json-stream/lib/json-stream.js:30:14)
    at Transform._read (_stream_transform.js:179:10)
    at Transform._write (_stream_transform.js:167:12)
    at doWrite (_stream_writable.js:226:10)
    at writeOrBuffer (_stream_writable.js:216:5)
    at Writable.write (_stream_writable.js:183:11)
    at write (_stream_readable.js:602:24)
    at flow (_stream_readable.js:611:7)
    at IncomingMessage.pipeOnReadable (_stream_readable.js:643:5)
    at IncomingMessage.emit (events.js:92:17)
    at emitReadable_ (_stream_readable.js:427:10)
    at emitReadable (_stream_readable.js:423:5)
    at readableAddChunk (_stream_readable.js:166:9)
    at IncomingMessage.Readable.push (_stream_readable.js:128:10)
    at HTTPParser.parserOnBody [as onBody] (http.js:173:22)
```

**Comment by knownasilya** [comment]
@HendrikRoth I think @oliversalzburg meant that it should be passed in there, but isn't. Hinting that a PR is needed 👍

**Comment by oliversalzburg** [comment]
Should be resolved with 0a25696a4622b85014d9cde0920f3e5b675653a8
